pdns/dnsdist-lua-bindings.cc

```diff
@@ -172,6 +174,29 @@ void setupLuaBindings(bool client)
 
   /* ComboAddress */
   g_lua.writeFunction("newCA", [](const std::string& name) { return ComboAddress(name); });
+  g_lua.writeFunction("newCAFromRaw", [](const std::string& raw, boost::optional<uint16_t> port) {
```

Comment:
Maybe document the new functions introduced here ?

Reply:
This function is very specific to what we do with our own Lua FFI code, I'll ponder if it's worth documenting it.